### PR TITLE
[nowide] update to 11.1.3

### LINKED ISSUE #21866: [nowide] update to 11.1.3
Library name: `nowide` (standalone Boost::nowide)

New version number: 11.1.3

Other information that may be useful (release notes, etc...)
<https://github.com/boostorg/nowide/releases/tag/v11.1.3>

### PR DESCRIPTION
**Pull request by tats-u**
**Describe the pull request**

- #### What does your PR fix?  
  Fixes #21866

- #### Which triplets are supported/not supported? Have you updated the [CI baseline](https://github.com/microsoft/vcpkg/blob/master/scripts/ci.baseline.txt)?  
  same as older versions

- #### Does your PR follow the [maintainer guide](https://github.com/microsoft/vcpkg/blob/master/docs/maintainers/maintainer-guide.md)?  
  Yes

- #### If you have added/updated a port: Have you run `./vcpkg x-add-version --all` and committed the result?  
  Yes

**If you are still working on the PR, open it as a Draft: https://github.blog/2019-02-14-introducing-draft-pull-requests/**

↑done

### CODE REVIEW

File: ports/nowide/portfile.cmake [hunk 1]
```diff
@@ -1,23 +1,22 @@
...
     SOURCE_PATH ${SOURCE_PATH}
-    PREFER_NINJA
     OPTIONS -DBUILD_TESTING=OFF
 )
 
-vcpkg_install_cmake()
+vcpkg_cmake_install()
 
-vcpkg_fixup_cmake_targets(CONFIG_PATH lib/cmake/nowide TARGET_PATH share/${PORT})
+vcpkg_cmake_config_fixup(CONFIG_PATH lib/cmake/nowide)
```
Comment (tats-u) [inline ports/nowide/portfile.cmake:19]: I wonder if the change is correct

Comment (autoantwort) [reply ports/nowide/portfile.cmake:19]: Yes it is correct 👍

**Comment by tats-u** [comment]
`Don't use deprecate functions anymore`
↓
`Don't use deprecated functions anymore`

**Comment by JonLiu1993** [comment]
@tats-u ,Thanks for your pr, If this pr is done, please let convert 'Draft' to 'Open'

**Comment by tats-u** [comment]
@JonLiu1993 I fixed the message and the branch is ready for merge.

File: ports/nowide/portfile.cmake [hunk 2]
```diff
@@ -1,23 +1,22 @@
...
 )
 
 vcpkg_extract_source_archive_ex(
     OUT_SOURCE_PATH SOURCE_PATH
     ARCHIVE ${ARCHIVE}
 )
 
-vcpkg_configure_cmake(
+vcpkg_cmake_configure(
     SOURCE_PATH ${SOURCE_PATH}
```
Comment (JonLiu1993) [inline ports/nowide/portfile.cmake:13]:
```suggestion
    SOURCE_PATH "${SOURCE_PATH}"
```

**Review by JonLiu1993 — CHANGES_REQUESTED** [review]

File: ports/nowide/portfile.cmake (hunk 2 above)
Comment (JonLiu1993) [reply ports/nowide/portfile.cmake:13]: @tats-u ,Thanks for your pr, we should add double quotes to the absolute path in the portfile.cmake like this

**Review by JonLiu1993 — APPROVED** [review]

**Review by BillyONeal — APPROVED** [review]

**Comment by BillyONeal** [comment]
Thanks for the update!

**Pull request merged**